src/main/java/org/kiwiproject/test/mongo/MongoServerTests.java

```diff
@@ -15,10 +16,23 @@
     /**
      * Start a new in-memory {@link MongoServer}.
      *
```

Comment:
nitpick: add a `<p>` here (highlighting and reformatting just the javadoc in IntelliJ will do it for you)